app/components/no_results.js

```diff
@@ -38,9 +38,9 @@ export default class NoResults extends PureComponent {
         } else if (iconName) {
             icon = (
                 <IonIcon
-                    size={44}
-                    color={changeOpacity(theme.centerChannelColor, 0.4)}
+                    size={128}
```

Comment:
I guess you could wrap this in a view to get the "gray" circle

Reply:
Done